mythril/laser/ethereum/instructions.py

```diff
@@ -895,7 +898,7 @@ def codesize_(self, global_state: GlobalState) -> List[GlobalState]:
         return [global_state]
 
     @StateTransition(enable_gas=False)
-    def sha3_(self, global_state: GlobalState) -> List[GlobalState]:
+    def sha3_(selxf, global_state: GlobalState) -> List[GlobalState]:
```

Comment:
what's this variable?

Reply:
Uhh.. funny that this didn't cause any errors.